Linker errors for _curl_easy_* from libgit2.a

Since 1b3f6aa3, I'm getting the following linker errors when building ObjectiveGit:

```
Undefined symbols for architecture x86_64:
  "_curl_easy_cleanup", referenced from:
      _curls_close in libgit2.a(curl_stream.c.o)
  "_curl_easy_getinfo", referenced from:
      _curls_connect in libgit2.a(curl_stream.c.o)
      _curls_certificate in libgit2.a(curl_stream.c.o)
  "_curl_easy_init", referenced from:
      _git_curl_stream_new in libgit2.a(curl_stream.c.o)
  "_curl_easy_perform", referenced from:
      _curls_connect in libgit2.a(curl_stream.c.o)
  "_curl_easy_recv", referenced from:
      _curls_read in libgit2.a(curl_stream.c.o)
  "_curl_easy_send", referenced from:
      _curls_write in libgit2.a(curl_stream.c.o)
  "_curl_easy_setopt", referenced from:
      _git_curl_stream_new in libgit2.a(curl_stream.c.o)
      _curls_set_proxy in libgit2.a(curl_stream.c.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

This happens in both Xcode 6.4 and 6.3.2 on OS X 10.10.4.

It's my fault for not catching this before merging #472.

--- a/ObjectiveGitFramework.xcodeproj/project.pbxproj
+++ b/ObjectiveGitFramework.xcodeproj/project.pbxproj
@@ -1467,6 +1467,7 @@
 					/usr/local/lib/libssh2.a,
 					"-lcrypto",
 					"-lssl",
+					"-lcurl",
 				);
 				PRODUCT_NAME = ObjectiveGit;
 				WRAPPER_EXTENSION = framework;
@@ -1490,6 +1491,7 @@
 					/usr/local/lib/libssh2.a,
 					"-lcrypto",
 					"-lssl",
+					"-lcurl",
 				);
 				PRODUCT_NAME = ObjectiveGit;
 				WRAPPER_EXTENSION = framework;
@@ -1677,6 +1679,7 @@
 					/usr/local/lib/libssh2.a,
 					"-lcrypto",
 					"-lssl",
+					"-lcurl",
 				);
 				PRODUCT_NAME = ObjectiveGit;
 				WRAPPER_EXTENSION = framework;
@@ -1899,6 +1902,7 @@
 					/usr/local/lib/libssh2.a,
 					"-lcrypto",
 					"-lssl",
+					"-lcurl",
 				);
 				PRODUCT_NAME = ObjectiveGit;
 				WRAPPER_EXTENSION = framework;

